swan/Dockerfile

```diff
@@ -64,9 +64,11 @@ RUN dnf -y install HEP_OSlibs-9.1.0-2.el9
 # Switch back to jovyan to avoid accidental container runs as root
 USER ${NB_UID}
 
+RUN pip install --no-deps --no-cache-dir \
```

Comment:
Not available in conda forge? Maybe document it so that we know why is not bellow.

Reply:
Done